### PR TITLE
Update ruby to 2.2.3 and all the other gems

### PR DESCRIPTION
**Pull request by matthewbentley**
to their latest versions

### CODE REVIEW

**Comment by matthewbentley** [comment]
I think I'm missing something.  Can anyone comment?

**Comment by matthewbentley** [comment]
Nevermind, .travis.yml needed updating.

File: Gemfile.lock [hunk 1]
```diff
@@ -348,3 +346,6 @@ DEPENDENCIES
   watir-webdriver
   web-console (~> 2.0)
   whenever
+
+BUNDLED WITH
```
Comment (ajm188) [inline Gemfile.lock]: can we remove these lines? bundler versions don't matter _that_ much and with this people won't be able to `bundle` unless they're using this exact version

Comment (matthewbentley) [inline Gemfile.lock]: I'm not sure what the proper way to do this is, so I just deleted the .lock file and re-ran bundle install, and this is what resulted.

Comment (ajm188) [inline Gemfile.lock]: You want `bundle update`. But I think that will still write the "BUNDLED WITH" bit. There might be a flag you can pass to bundler to make it not do that, but my current solution has been to just `git add -p Gemfile.lock` and not include that chunk.

**Comment by ajm188** [comment]
:+1:

**Comment by JKobyP** [comment]
Does this call for an update to the 'Contributing' section of the wiki?

**Comment by matthewbentley** [comment]
Probably a good idea

**Pull request merged**